starter_code/hangman.js

```diff
@@ -1,44 +1,124 @@
 let hangman;
 
-// class Hangman {
-//   constructor() {
+ class Hangman {
+   constructor() {
+  this.words=['jorge','ironhack','garibaldi','pokemon','gargola'];
+  this.secretWord = '';
+  this.letters=[];
+  this.guessedLetter='';
+  this.errorsLeft=10;
+   }
+
+ getWord() {
+   return this.words[Math.floor(Math.random() *this.words.length)];
+   }
+
+  checkIfLetter(keyCode) {
+    //65 corresponde al menor número que puede generar una letra que equivale a "a"
+    //90 corresponde al mayor numero que puede generar una letra que equivale a "z"
+   if(keyCode>=65&&keyCode<=90)  
+    return true;
+    return false;
+  }
+
+   checkClickedLetters(key) {
+   if(this.letters.includes(key))
+   return false
+   return true
+   }
+
+  addCorrectLetter(i) {
+  //  this.guessedLetter += this.secretWord[i].toUpperCase(); 
+    this.guessedLetter+=key;
```

Comment:
aqui andabas por buen camino y en realidad esto te funciona pero para pasar los test de jasmine the hacian falta algunas condiciones, este codigo funciona y te pasa los test en verde 

    `let newStr = this.secretWord.split("")`
 `   this.guessedLetter += newStr[i].toUpperCase()`